modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -0,0 +1,183 @@
+ifndef::env-kubernetes[. <<Start a target cluster>> (new cluster).]
+ifndef::env-kubernetes[. <<restore-to-target-cluster, Restore data from a failed source cluster to the new cluster>>.]
+ifndef::env-kubernetes[. <<Perform a rolling restart>>.]
+ifdef::env-kubernetes[. <<Verify that the cluster restore is complete>>.]
+
+=== Prerequisites
+
+You must have the following:
+
+- Tiered Storage enabled on the source cluster.
+- Physical or virtual machines on which to deploy the target cluster.
+
+=== Limitations
+
+- Whole cluster restore supports only one source cluster. It is not possible to consolidate multiple clusters onto the target cluster.
+
+- If a duplicate cluster configuration is found in the target cluster, it will be overwritten by the restore.
+
+- The target cluster should not contain user-managed or application-managed topic data, schemas, users, ACLs, or ongoing transactions.
+
+=== Start a target cluster
+
+ifdef::env-kubernetes[]
+
+Deploy the target Redpanda cluster.
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    storage:
+      tiered:
+        <tiered-storage-settings>
+    config:
+      cluster:
+        cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`cluster-restore.yaml`
+[,yaml]
+----
+storage:
+  tiered:
+    <tiered-storage-settings>
+config:
+  cluster:
+    cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values cluster-restore.yaml
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set storage.tiered.<tiered-storage-settings> \
+  --set config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap=true
+```
+====
+--
+======
+
+- `storage.tiered`: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+- `config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap`: Automate cluster restore in Kubernetes. Setting to `true` is recommended when using an automated method for deployment. When bootstrapping a cluster with a given bucket, make sure that any previous cluster using the bucket is fully destroyed; otherwise Tiered Storage subsystems may interfere with each other.
+endif::[]
+
+ifndef::env-kubernetes[]
+For self-hosted, follow the steps to xref:deploy:deployment-option/self-hosted/manual/index.adoc[deploy a new cluster].
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster].
```

Comment:
@kbatuigas  I'm not sure we should be saying anything about cloud as cloud is seeking to automate this on behalf of customers, and for Dedicated im not sure customers even have direct access to the credentials required to restore.  @bpraseed  shouldn't we exclude cloud from Whole Cluster Restore docs for now?

Reply:
Praseed confirmed on Slack that we will control this.